### DISCUSSION TITLE
[Question][Hw3] No newline at end of file (example3.expected)

### DISCUSSION THREAD

**Discussion by Standchen** [prosyslab-classroom/cs348-information-security#528] [Q&A]
![image](https://user-images.githubusercontent.com/68227020/116540780-91cdcd80-a925-11eb-8c82-e3e04aba36d2.png)

I found that the content of `example3.expected` is `Divide-by-zero detected at line 11 and col 15`, **without ending with newline character**.
But considering the line 8 of `runtime.c`(`printf("Divide-by-zero detected at line %d and col %d\n", line, col);`), I think it should end with newline character, as other expected files (`example1.expected` and `example2.expected`) do.

Please let me know if it is intended, or I just overlooked something.

Thank you.

**Comment by Junseon00** [comment]
Yes, I'm fixing it now. I'll let you know after we commit the fixed code to origin repo. 
Please didn't close this issue until we announced about it :) Thanks.
